adapter/config/ingress/status.go

```diff
@@ -68,14 +71,21 @@ func NewStatusSyncer(mesh *proxyconfig.ProxyMeshConfig, client kubernetes.Interf
 	if mesh.IngressService != "" {
 		publishService = fmt.Sprintf("%v/%v", options.Namespace, mesh.IngressService)
 	}
+	glog.Infof("INGRESS STATUS publishService ", publishService)
```

Comment:
The files related to the go build will disappear after the merge.

The other logs I found important during troubleshooting, and I think we should leave them in. I generally used V(3).